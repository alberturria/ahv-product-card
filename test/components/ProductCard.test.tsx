import React from 'react';
import renderer from 'react-test-renderer';
import ProductCard from '../../src/components';
import { product1 } from '../data/products';

describe('ProductCard', () => {
  test('Should have render the component properly', () => {
    const wrapper = renderer.create(
      <ProductCard product={product1}>{() => <h1>ProductCard</h1>}</ProductCard>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
