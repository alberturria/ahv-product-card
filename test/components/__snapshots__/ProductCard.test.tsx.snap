// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCard Should have render the component properly 1`] = `
<div
  className="productCard undefined"
>
  <h1>
    ProductCard
  </h1>
</div>
`;

exports[`ProductCard Should've increased the counter 1`] = `
<div
  className="productCard undefined"
>
  <h1>
     ProductCard
  </h1>
  <span>
    0
  </span>
  <button
    onClick={[Function]}
  />
</div>
`;
